lib/ButtonNew/ButtonBase.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import cx from 'classnames';
+import { string, arrayOf } from 'prop-types';
+import Icon from '../Icon';
+import { basePropTypes, baseDefaultProps, sizes } from './common';
+
+const getSizeBasedClasses = size => {
+  const { sm, md } = sizes;
+
+  if (size === md) {
+    return 'text-base py-4 px-5 m-5';
+  }
+
+  if (size === sm) {
+    return 'text-base p-3 m-3';
+  }
+
+  return 'text-sm py-1 px-3 m-3';
```

Comment:
🚩How come this isnt with the rest of our classes? I'm assuming there's a default prop for size that we can use for the default